src/js/views/labels_from_maps_modal.js

```diff
@@ -85,7 +85,7 @@ var LabelFromMapsModal = Backbone.View.extend({
             if (imageKeyValues[iid]) {
                 var labels = imageKeyValues[iid].map(function(value){
                     return {
-                        'text': includeKey ? (value[0].replaceAll("_","\\_") + ': ' + value[1].replaceAll("_","\\_")) : value[1].replaceAll("_","\\_"),
+                        'text': includeKey ? (key.replaceAll("_","\\_") + ': ' + value.replaceAll("_","\\_")) : value.replaceAll("_","\\_"),
```

Comment:
I think that in fixing those merge conflicts you've got an error here. `value` is actually a list of `[key, value]`. It actually looks like you don't need any change to this line, since you've already got the escaping logic there?

Reply:
True there was an error here. I didn't notice it before. Thanks